Antenna_Performance_Metric/rootAnalysis.py

```diff
@@ -197,6 +191,11 @@ def EffectiveVolume2(thisColor,thisLabel):
 
 def AddErrors(all_weights):
     bin_num = 10
+    
+    # If no events pass, just return 0's
+    if len(all_weights) == 0:
+        return(0,0)
```

Comment:
```suggestion
    if not all_weights:
        return 0, 0
```
Empty lists are considered false by default, so you don't need to use `len()`.
In case you want to keep it to be more explicit, use this suggestion instead:
```suggestion
    if len(all_weights) == 0:
        return 0, 0
```